xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.adm.sudo"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>sudo <!-- TODO: Better title --></title>
```

Comment:
sudo is fine as a title, though you could put it in command. I would leave away the subtitle -- does not fit into our style of documentation.